mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -596,16 +596,7 @@ private void doRenameMailbox(Mailbox mailbox, MailboxPath newMailboxPath, Mailbo
     }
 
     private List<MailboxMetaData> searchMailboxes(MailboxQuery mailboxQuery, MailboxSession session, Right right) throws MailboxException {
```

Comment:
please rename this one as `searchMailboxMetadata` and rename `searchRowMailboxes` into `searchMailboxes`